### ISSUE TITLE
Improve expanded need view

### ISSUE THREAD

**Issue by fkleedorfer** [researchstudio-sat/webofneeds#1880]
In order to alleviate the problems discussed in #1742, I suggest these changes:
[ ] There is a horizontal line that appears when a need is expanded in the list view. That line makes it look like the lower part does not belong to the need. Get rid of that line.
[ ] The 'Show Post Details' button, being sometimes shown below the match/message icons is confusing and bulky.  When below the buttons, it seems to belong to the matches, especially when there is only one match, rather than to the need. Solution: shorten label to 'Details' and put it on the same line with the buttons.

**Comment by quasarchimaere** [comment]
"show details" button -> moved above the indicators, since even with the one word label we might not have enough space to display it in line with the indicators and texts...

for the horizontal line i would refer to the ux specialists because leaving out the first border will look pretty horrible -> see screenshot

**Comment by fkleedorfer** [comment]
No the line I wanted removed was the other one, in your screenshot: between 'What's Around' and (button) 'No Chats or Requests' (button) '20 Matches - 20 new etc

**Comment by quasarchimaere** [comment]
@fkleedorfer oh ok, well i do not want to get rid of that line for styling purposes, its not a full line anyway it creates a distinction between the visible needcontent and the indicators/actions

**Comment by fkleedorfer** [comment]
that's what I mean - that distinction looks like it's the end of the need. it's hard to parse visually
